packages/debugger/style/variables.css

```diff
@@ -114,6 +114,10 @@
   color: var(--jp-content-font-color0);
 }
 
+.jp-VariableRenderer {
+  overflow: auto;
+}
+
```

Comment:
```suggestion
.jp-VariableRendererPanel {
  overflow: auto;
}

.jp-VariableRendererPanel-renderer {
  overflow: auto;
  height: 100%
}

```

Without adding height 100% to the child widget, the scrollbar gets positioned in the middle of the panel:

![Screenshot from 2022-08-19 14-10-59](https://user-images.githubusercontent.com/5832902/185625567-a1ee8335-1d4b-4ce9-a218-a6a9f037f2d2.png)

with:

![Screenshot from 2022-08-19 14-11-03](https://user-images.githubusercontent.com/5832902/185625542-0febb022-66bc-47b3-86f0-0140679e207c.png)

Reply:
I was looking at the same time to this PR. Simply using `visible` for the variable view seems enough (at least on Firefox).

```suggestion
.jp-VariableRendererPanel {
  overflow: auto;
}
.jp-VariableRendererPanel-renderer {
  overflow: visible;
}

```